client-core/src/wallet/default_wallet_client.rs

```diff
@@ -1066,6 +1066,15 @@ where
     /// import a plain base64 encoded plain transaction
     fn import_plain_tx(&self, name: &str, enckey: &SecKey, tx_str: &str) -> Result<Coin> {
         let tx_info = TransactionInfo::decode(tx_str)?;
+
+        let found_tx = self.export_plain_tx(name, enckey, &hex::encode(tx_info.tx.id()));
```

Comment:
`export_plain_tx` is slightly overkill for checking transaction existence. Maybe a method to get only/check existence of the transaction from storage?

Reply:
ok i will modifiy it